# Clase 1

## Operaciones básicas

### Suma o adición ($+$)

In [1]:
47896+83924+1785

133605

\begin{align*}
47896&\\
+\ 83924&\\
\underline{\phantom{00}1785}&\\
133605&
\end{align*}

### Resta, diferencia o sustracción ($-$)

In [2]:
987654-234567

753087

\begin{align*}
987654&\\
-\ \underline{234567}&\\
753087&
\end{align*}

### Multiplicación o producto ($\times$)

In [3]:
4137*745

3082065

\begin{align*}
4137&\\
\underline{\times\ 745}&\\
20685&\\
16548\phantom{0}&\\
\underline{28959\phantom{00}}&\\
3082065&
\end{align*}

### División o cociente ($\div$)

\begin{align*}
\phantom{120|07}629&\\
12\ \overline{|\ 7549}&\\
\underline{-72}\phantom{00}&\\
34\phantom{0}&\\
\underline{-24}\phantom{0}&\\
109&\\
\underline{-108}&\\
1
\end{align*}

Al número que se encuentra arriba de la "casita" se le conoce como ***cociente***; al número que se encuentra a la izquierda de la "casita" se le conoce como ***divisor***; al número que se encuentra dentro de la "casita" se le conoce como ***dividendo***; al número que se encuentra hasta abajo del proceso de división se le conoce como ***residuo***.

$$\dfrac{7549}{12}=629+\dfrac{1}{12}$$

### Potencia

$$ 3^4=3\cdot3\cdot3\cdot3=81 $$

$$ 5^3=5\cdot5\cdot5=125 $$

*El resultado de $3$ a la $4$ es equivalente a multiplicar el $3$ por sí mismo $4$ veces.*

Al número que se encuentra en la posición del $3$ se le conoce como ***base***; al número que se encuentra arriba, a la derecha del $3$, se le conoce como ***exponente***; el resultado de esta operación se llama ***potencia***.

*El resultado de una potencia es equivalente a multiplicar la base por sí misma el número del exponente veces.*

### Raíz o radicación ($\sqrt{\phantom{0}}$)

$$ \sqrt[4]{81}=3 $$

$$ \sqrt[4]{625}=5 $$

*El resultado de raíz cuártica de $81$ es equivalente a preguntarse sobre el número que multiplicado por sí mismo $4$ veces es $81$.*

Al número que se encuentra arriba de la raíz, el $3$ en el caso anterior, se le conoce como ***radical***.

$$ \sqrt[n]{a}=a^{1/n} $$

$$ 8^{1/3}=\sqrt[3]{8} $$

$$ 15^{1/4}=\sqrt[4]{15} $$

## Operaciones básicas entre signos

\begin{align*}
(+)(+)=(+)\quad&\quad(+)\div(+)=(+)\\
(-)(-)=(+)\quad&\quad(-)\div(-)=(+)\\
(+)(-)=(-)\quad&\quad(+)\div(-)=(-)\\
(-)(+)=(-)\quad&\quad(-)\div(+)=(-)
\end{align*}

Entonces, obtenemos el mismo tipo de signo si multiplicamos o dividimos.

## Operaciones básicas con números positivos y negativos

* $1\rightarrow\cap$
* $0\rightarrow-$
* $-1\rightarrow\cup$

* ***Ejemplo:***

$$ 1+(-1)=0 $$

$$ (\cap)+(\cup)=- $$

* ***Ejemplo:***

$$3+4=7$$

* ***Ejemplo:***

$$3+(-2)=3-2=1$$

* ***Ejemplo:***

$$3+(-3)=3-3=0 $$

* ***Ejemplo:***

$$1+(-3)=-2$$

* ***Ejemplo:***

$$-2+(-3)=-2-3=-5$$

* ***Ejemplo:***

$$4+(-7)=-3 $$

*Siempre que un número no tenga el signo explícitamente, se entiende que es positivo de manera implícita.*

* ***Ejemplo:***

$$-4+(-7)=-11 $$

## Operaciones básicas con fracciones

\begin{equation*}
\dfrac{a}{b}\ {{\leftarrow\ \text{numerador}\phantom{or}}\atop{\leftarrow\ \text{denominador}}}
\end{equation*}

* ***denominador*** porque denomina en cuántas partes partimos el pastel
* ***numerador*** porque denomina cuántas partes tomamos (nos vamos a comer).

### Simplificación y complicación de fracciones

Una fracción es igual a otra si se multiplica el numerador y el denominador por un mismo número.

* ***Ejemplo:***

$$ \dfrac{2}{9}=\dfrac{8}{36} $$

Una fracción es igual a otra si se divide el numerador y el denominador por un mismo número.

* ***Ejemplo:***

$$ \dfrac{144}{256}=\dfrac{72}{128}=\dfrac{36}{64}=\dfrac{9}{16} $$

* ***Ejemplo:***

$$ \dfrac{140}{98}=\dfrac{70}{49}=\dfrac{10}{7} $$

* ***Ejemplo:***

$$ \dfrac{10}{7}=\dfrac{10\cdot14}{7\cdot14}=\dfrac{140}{98} $$

### Suma o adición ($+$)

$$ \dfrac{a}{b}+\dfrac{c}{d}=\dfrac{ad+bc}{bd} $$

**Nota importante:** para poder sumar o restar los numeradores de dos fracciones directamente, es necesario que tengan el mismo denominador.

Si quieren ver otra forma de hacer suma de fracciones, les recominendo revisar este vídeo corto:

https://youtu.be/MQFTAGdVWjo

* ***Ejemplo:***

$$ \dfrac{2}{7}+\dfrac{4}{7}=\dfrac{2+4}{7}=\dfrac{6}{7} $$

* ***Ejemplo:***

$$ \dfrac{3}{5}+\dfrac{1}{2}=\dfrac{3\cdot2+5\cdot1}{5\cdot2}=\dfrac{11}{10} $$

* ***Ejemplo:***

$$ \dfrac{9}{7}+\dfrac{8}{11}=\dfrac{99+56}{77}=\dfrac{155}{77} $$

### Resta, diferencia o sustracción ($-$)

$$ \dfrac{a}{b}+\left(-\dfrac{c}{d}\right)=\dfrac{ad-bc}{bd} $$

Si quieren ver otra forma de hacer resta de fracciones, les recomiendo revisar este vídeo corto:

https://youtu.be/_-piXiAuRT8

* ***Ejemplo:***

$$ \dfrac{3}{5}-\dfrac{1}{2}=\dfrac{3\cdot2-5\cdot1}{5\cdot2}=\dfrac{1}{10} $$

* ***Ejemplo:***

$$ \dfrac{9}{7}-\dfrac{8}{11}=\dfrac{99-56}{77}=\dfrac{43}{77} $$

* ***Ejemplo:***

$$ \dfrac{1}{2}-\dfrac{2}{3}=\dfrac{3-4}{6}=\dfrac{-1}{6}=-\dfrac{1}{6} $$

* ***Ejemplo:***

$$ -\dfrac{3}{7}+\dfrac{1}{9}=\dfrac{-20}{63}=-\dfrac{20}{63} $$

* ***Ejemplo:***

$$ -\dfrac{3}{4}-\dfrac{5}{8}=\dfrac{-44}{32}=-\dfrac{44}{32}=-\dfrac{22}{16}=-\dfrac{11}{8} $$

1. Siempre que tengas signos iguales, suma los números.
2. Después, agrega el signo al resultado final.

### Multiplicación o producto ($\times$)

Esto es directo: el resultado el producto de los numeradores en el numerador y el producto de los denominadores en el denominador.

$$ \dfrac{a}{b}\cdot\dfrac{c}{d}=\dfrac{ac}{bd} $$

Si quieren ver otra forma de hacer resta de fracciones, les recomiendo revisar este vídeo corto:

https://youtu.be/_-piXiAuRT8

* ***Ejemplo:***

$$ \dfrac{3}{5}\cdot\dfrac{10}{81}=$$

### División o cociente ($\div$)

$$ \dfrac{a}{b}\div\dfrac{c}{d}=\dfrac{\dfrac{a}{b}}{\dfrac{c}{d}}=\dfrac{ad}{bc} $$

$$ \dfrac{a}{b}\div\dfrac{c}{d}=\dfrac{a}{b}\cdot\dfrac{d}{c}=\dfrac{ad}{bc} $$

Si quieren ver otra forma de hacer división de fracciones, les recomiendo revisar este vídeo corto:

https://youtu.be/o4xoVTKRpgo